docs/axes/labelling.md

```diff
@@ -9,6 +9,7 @@ The scale label configuration is nested under the scale configuration in the `sc
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | `display` | `boolean` | `false` | If true, display the axis title.
+| `labelAlign` | `string` | `center` | Alignment of the axis title. Possible options are `start`, `center` and `end`
```

Comment:
Strings should be in single quotes. I.e. `'center'`, etc.

Reply:
Fixed and pushed the changes